lib/main_common.pm

```diff
@@ -533,9 +533,11 @@ sub load_extra_tests {
         # well
         if (check_var('DESKTOP', 'gnome')) {
             loadtest 'x11/yast2_lan_restart';
-            loadtest 'x11/network/hwsim_wpa2_enterprise_setup';
-            loadtest 'x11/network/yast2_network_setup';
-            loadtest 'x11/network/NM_wpa2_enterprise';
+            if (!(is_sle && !sle_version_at_least('12-SP3'))) {
```

Comment:
for the untrained eye: this is !is_sle || sle_version_at_least('12-SP3')

Reply:
yeah, right